src/ui/MobileSearch/Search.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import Header from './Header.svelte'
+  import Filter from './Filter.svelte'
+  import Assets from './Assets.svelte'
+  import Trends from './Trends.svelte'
+  import Insights from './Insights.svelte'
+  import People from './People.svelte'
+  import { FILTER_ITEMS } from './utils'
+
+  export let show = true
+
+  let searchTerm = ''
+  let filter = FILTER_ITEMS.Assets
+</script>
+
+{#if show}
+  <section class="relative">
+    <Header bind:searchTerm bind:show />
+    <Filter bind:filter />
+    {#if filter === FILTER_ITEMS.Assets}
+      <Assets {searchTerm} {filter} />
+    {:else if filter === FILTER_ITEMS.Trends}
+      <Trends {searchTerm} {filter} />
+    {:else if filter === FILTER_ITEMS.Insights}
+      <Insights {searchTerm} {filter} />
+    {:else if filter === FILTER_ITEMS.People}
+      <People {searchTerm} {filter} />
+    {/if}
```

Comment:
Instead of duplicating code for each component (Assets, Trends, ...etc), keep the main logic inside `src/ui/MobileSearch/Suggestions.svelte`. 

Selected tab should pass the:
- `query` - function, used for fetching the tab's data;
- `filter` - function, used for filtering out data; 
-  `Component` - component, used for rendering an item; Includes only item's layout and styles. Accepts `item` prop.

Suggested tabs' scheme:
```js
{
  title: 'Assets',
  query: queryProjects,
  filter: (searchTerm, items) => items.filter(({ name, ticker }) => {
       return name.toLowerCase().includes(searchTerm) || ticker.toLowerCase().includes(searchTerm)
     }),
  Component: Asset,
}
```

`src/ui/MobileSearch/Suggestions.svelte` will dynamically render item's component using
```svelte
<svelte:component this={Component} {item} />
```